api/v1alpha1/snapshotpod_types.go

```diff
@@ -57,6 +57,15 @@ type ImageSaveOptions struct {
 	// tag - the tag of origin image, if not set, will be latest, if use sha format image like nginx@sha256:abcdef, tag will be the real sha value like abcdef
 	ImageRefFormat    string `json:"imageRefFormat,omitempty"`
 	RegistrySecretRef string `json:"registrySecretRef,omitempty"`
+	// MaxRetries defines the maximum number of retries when the task fails
+	// +optional
+	// +kubebuilder:default:=3
```

Comment:
According to the definition of this comment, a total of 1+3=4 will be executed. Is this in line with expectations?